packages/grid/x-data-grid-premium/src/hooks/features/aggregation/useGridAggregationPreProcessors.tsx

```diff
@@ -0,0 +1,189 @@
+import * as React from 'react';
+import MuiDivider from '@mui/material/Divider';
+import { gridColumnLookupSelector } from '@mui/x-data-grid-pro';
+import {
+  GridPipeProcessor,
+  GridRestoreStatePreProcessingContext,
+  GridRowTreeCreationValue,
+  useGridRegisterPipeProcessor,
+} from '@mui/x-data-grid-pro/internals';
+import { GridApiPremium } from '../../../models/gridApiPremium';
+import {
+  getAvailableAggregationFunctions,
+  addFooterRows,
+  getAggregationRules,
+  mergeStateWithAggregationModel,
+} from './gridAggregationUtils';
+import {
+  wrapColumnWithAggregationValue,
+  unwrapColumnFromAggregation,
+} from './wrapColumnWithAggregation';
+import { DataGridPremiumProcessedProps } from '../../../models/dataGridPremiumProps';
+import { GridAggregationColumnMenuItem } from '../../../components/GridAggregationColumnMenuItem';
+import { gridAggregationModelSelector } from './gridAggregationSelectors';
+import { GridInitialStatePremium } from '../../../models/gridStatePremium';
+import { GridAggregationRules } from './gridAggregationInterfaces';
+
+const Divider = () => <MuiDivider onClick={(event) => event.stopPropagation()} />;
```

Comment:
Why `event.stopPropagation()` is needed here?

Reply:
We have the same on column pinning
Maybe @m4theushw can answer

Reply:
I added it to not sort when clicking the divider. 😁